test/shell_runner.go

```diff
@@ -90,7 +91,13 @@ func (runner *ShellRunner) Run(name string, arguments ...string) (output string,
 	cmd.Dir = runner.dir
 	cmd.Env = customEnv
 	rawOutput, err := cmd.CombinedOutput()
-	return string(rawOutput), err
+	output = string(rawOutput)
+	if Debug {
+		fmt.Printf("In %q:\n", path.Base(runner.dir))
```

Comment:
Thoughts on using the format we used in ruby where the dir shares the line with the command?
```
dir > name args...
```
https://github.com/Originate/git-town/blob/master/features/support/run_helpers.rb#L48

Reply:
Good idea, done.